src/wings_ko.lang

```diff
@@ -1994,15 +1994,15 @@
    [
     % "Light ~ts"
     {1,"라이트 ~ts"},
-    % ": Pos ~ts ~ts ~ts"
-    {2,": Pos ~ts ~ts ~ts"}
+    % ": Pos ~s"
+    {2,": Pos ~s"}
    ]},
   {info_1,
    [
-    % ". Aim ~ts ~ts ~ts. Dir ~ts ~ts ~ts"
-    {1,". Aim ~ts ~ts ~ts. Dir ~ts ~ts ~ts"},
-    % ". Angle ~ts~c"
-    {2,". 각도 ~ts~c"}
+    % ". Aim ~s. Dir ~s"
+    {1,". Aim ~s. Dir ~s"},
+    % ". Angle ~s~c"
+    {2,". 각도 ~t~c"}
```

Comment:
~t ?

Reply:
Yeah, tkbd already advised me about that. It looks like I removed "s" instead of "t". I need to review them. 
I'm going to check all the lang files for the _info_ and _info_1_